### DISCUSSION TITLE
Age data for 2021 NBS Vesteraalen pollock have been added to RACEBASE

### DISCUSSION THREAD

**Discussion by NancyRoberson** [afsc-gap-products/model-based-indices#47] [General]
Hi Mod Squad,

FYI, age data have been added to RACEBASE for 2021 NBS Vesteraalen pollock.  n= 192.  It is the full set and not a subset of the specimens collected.

Thanks!
Nancy

**Comment by Lewis-Barnett-NOAA** [comment]
Thanks Nancy!

Jim - New age data just came in for pollock.  These were not incorporated
into the hindcast ALK.  I just wanted to verify what you want for the
production runs this fall...I'm assuming we want to incorporate all
complete years in the ALK (or is there another consideration for how you
want to incorporate new NBS data specifically?)?  Do you also want another
run with the old ALK and new lengths to bridge?

Lewis


On Fri, Jun 9, 2023 at 7:32 AM Nancy Roberson ***@***.***>
wrote:

> Hi Mod Squad,
>
> FYI, age data have been added to RACEBASE for 2021 NBS Vesteraalen
> pollock. n= 192. It is the full set and not a subset of the specimens
> collected.
>
> Thanks!
> Nancy
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/afsc-gap-products/model-based-indices/discussions/47>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAKMJPYJ7OUQHETFQ3DV3TTXKMXV7ANCNFSM6AAAAAAZAXNJNA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Lewis Barnett, PhD (he/him/his)
Research Fish Biologist

NOAA Fisheries, Alaska Fisheries Science Center
7600 Sand Point Way NE, Bldg 4
Seattle, Washington 98115
Google Voice: (206) 526-4111

**Comment by Lewis-Barnett-NOAA** [comment]
Hi, sorry for the late reply, comments below

On Fri, Jun 9, 2023 at 8:35 AM Lewis Barnett - NOAA Federal <
***@***.***> wrote:

> Thanks Nancy!
>
> Jim - New age data just came in for pollock.  These were not incorporated
> into the hindcast ALK.  I just wanted to verify what you want for the
> production runs this fall...I'm assuming we want to incorporate all
> complete years in the ALK (or is there another consideration for how you
> want to incorporate new NBS data specifically?)?  Do you also want another
> run with the old ALK and new lengths to bridge?
>

Ideally it would be great to see the 2021 data updated and run to compare
with how much it changes the 2021 values I presently use. Historically,
Stan would make a couple of different design-based versions (i.e., with 2
ALKs vs 1 ALK). It'd be interesting to have runs for the historical data
done this way and perhaps adding in a third version w/ 3 ALKs (the third
being NBS all on its own). That seems like step one (but if the 3-ALK
version is too complicated it's probably ok to omit).

Then we'd need to decide how to fold in the new NBS data into the VAST
modeled age-composition estimates (I guess that should be straightforward
since it's just adding in more data that's spatially resolved). It would
also be cool to compare how they would come out.

As far as production runs in the fall, I'm unsure the age-growth group can
handle getting more samples done in the current year (they've already
requested reductions from past requests). So the additions may have to
occur as resources become available within their group. The results from
the outline above could help guide prioritizations in this regard.

Cheers!
Jim



>
> Lewis
>
>
> On Fri, Jun 9, 2023 at 7:32 AM Nancy Roberson ***@***.***>
> wrote:
>
>> Hi Mod Squad,
>>
>> FYI, age data have been added to RACEBASE for 2021 NBS Vesteraalen
>> pollock. n= 192. It is the full set and not a subset of the specimens
>> collected.
>>
>> Thanks!
>> Nancy
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/afsc-gap-products/model-based-indices/discussions/47>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AAKMJPYJ7OUQHETFQ3DV3TTXKMXV7ANCNFSM6AAAAAAZAXNJNA>
>> .
>> You are receiving this because you are subscribed to this thread.Message
>> ID: ***@***.***
>> com>
>>
>
>
> --
> Lewis Barnett, PhD (he/him/his)
> Research Fish Biologist
>
> NOAA Fisheries, Alaska Fisheries Science Center
> 7600 Sand Point Way NE, Bldg 4
> Seattle, Washington 98115
> Google Voice: (206) 526-4111
>


-- 

James Ianelli
NMFS/NOAA Building 4
7600 Sand Pt Way NE
Seattle WA 98115

206 526-6510
206 679-6674

See the special issue on natural mortality in fisheries assessments
<https://www.sciencedirect.com/journal/fisheries-research/special-issue/10NSQ74ZXD9>

Visit the ADMB project http://admb-project.org/

Also see Alaska groundfish stock assessments
<https://www.fisheries.noaa.gov/alaska/population-assessments/north-pacific-groundfish-stock-assessment-and-fishery-evaluation>

**Comment by Lewis-Barnett-NOAA** [comment]
Gotcha Jim, thanks for weighing in.  We can start with examining the
differences in the age-length key itself (and the design-based comps).  As
you noted, it will be straightforward to include the new ALK in the VAST
comps, but certainly more time-consuming to complete all iterations, so
we'll check to see how much the ALK changes first.

Lewis

On Tue, Jun 13, 2023 at 9:22 AM Jim Ianelli - NOAA Federal <
***@***.***> wrote:

>
> Hi, sorry for the late reply, comments below
>
> On Fri, Jun 9, 2023 at 8:35 AM Lewis Barnett - NOAA Federal <
> ***@***.***> wrote:
>
>> Thanks Nancy!
>>
>> Jim - New age data just came in for pollock.  These were not incorporated
>> into the hindcast ALK.  I just wanted to verify what you want for the
>> production runs this fall...I'm assuming we want to incorporate all
>> complete years in the ALK (or is there another consideration for how you
>> want to incorporate new NBS data specifically?)?  Do you also want another
>> run with the old ALK and new lengths to bridge?
>>
>
> Ideally it would be great to see the 2021 data updated and run to compare
> with how much it changes the 2021 values I presently use. Historically,
> Stan would make a couple of different design-based versions (i.e., with 2
> ALKs vs 1 ALK). It'd be interesting to have runs for the historical data
> done this way and perhaps adding in a third version w/ 3 ALKs (the third
> being NBS all on its own). That seems like step one (but if the 3-ALK
> version is too complicated it's probably ok to omit).
>
> Then we'd need to decide how to fold in the new NBS data into the VAST
> modeled age-composition estimates (I guess that should be straightforward
> since it's just adding in more data that's spatially resolved). It would
> also be cool to compare how they would come out.
>
> As far as production runs in the fall, I'm unsure the age-growth group can
> handle getting more samples done in the current year (they've already
> requested reductions from past requests). So the additions may have to
> occur as resources become available within their group. The results from
> the outline above could help guide prioritizations in this regard.
>
> Cheers!
> Jim
>
>
>
>>
>> Lewis
>>
>>
>> On Fri, Jun 9, 2023 at 7:32 AM Nancy Roberson ***@***.***>
>> wrote:
>>
>>> Hi Mod Squad,
>>>
>>> FYI, age data have been added to RACEBASE for 2021 NBS Vesteraalen
>>> pollock. n= 192. It is the full set and not a subset of the specimens
>>> collected.
>>>
>>> Thanks!
>>> Nancy
>>>
>>> —
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/afsc-gap-products/model-based-indices/discussions/47>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/AAKMJPYJ7OUQHETFQ3DV3TTXKMXV7ANCNFSM6AAAAAAZAXNJNA>
>>> .
>>> You are receiving this because you are subscribed to this thread.Message
>>> ID: ***@***.***
>>> com>
>>>
>>
>>
>> --
>> Lewis Barnett, PhD (he/him/his)
>> Research Fish Biologist
>>
>> NOAA Fisheries, Alaska Fisheries Science Center
>> 7600 Sand Point Way NE, Bldg 4
>> Seattle, Washington 98115
>> Google Voice: (206) 526-4111
>>
>
>
> --
>
> James Ianelli
> NMFS/NOAA Building 4
> 7600 Sand Pt Way NE
> Seattle WA 98115
>
> 206 526-6510
> 206 679-6674
>
> See the special issue on natural mortality in fisheries assessments
> <https://www.sciencedirect.com/journal/fisheries-research/special-issue/10NSQ74ZXD9>
>
> Visit the ADMB project http://admb-project.org/
>
> Also see Alaska groundfish stock assessments
> <https://www.fisheries.noaa.gov/alaska/population-assessments/north-pacific-groundfish-stock-assessment-and-fishery-evaluation>
>
>

-- 
Lewis Barnett, PhD (he/him/his)
Research Fish Biologist

NOAA Fisheries, Alaska Fisheries Science Center
7600 Sand Point Way NE, Bldg 4
Seattle, Washington 98115
Google Voice: (206) 526-4111

**Comment by Lewis-Barnett-NOAA** [comment]
Hi All,

I have heard the data will be finalized within the next few days to a
week.  This is a reminder to plan to start running production indices/comps
as soon as possible after the data are in.

As another reminder, those running models of age composition should
determine whether there are updated ages that have been processed that
should be incorporated into the ALK (based on issues posted by Nancy
Roberson:
https://github.com/afsc-gap-products/model-based-indices/discussions/46).
If there are new data going into an ALK, be sure to compare the resulting
new ALK to the prior ALK to ensure that any differences are relatively
minor.

For the Bering data, once we get the word that the data are finalized, we
need to make some tables accessible to sumfish by running this
function: sumfish::updateSurvey() ...or otherwise obtain the precursor data
from gapindex (but I think not quite everything is ready there? Zack let us
know if we can make the transition this production cycle or to hold off).
I may be out when the data are finalized, so in that case I would
appreciate it if somebody else with sumfish installed could run that and
let folks know it is complete before running the NBS data prep code in the
model-based-index repo.

If you have any questions or just want to look through outputs together,
please let me know and/or find a time on my calendar.

Thanks!
Lewis

-- 
Lewis Barnett, PhD (he/him/his)
Research Fish Biologist

NOAA Fisheries, Alaska Fisheries Science Center
7600 Sand Point Way NE, Bldg 4
Seattle, Washington 98115
Google Voice: (206) 526-4111
